### PR TITLE
feat: Add support to producer end-to-end encryption

### PR DESCRIPTION
**Pull request by jffp113**
I added support for pulsar end-to-end encryption.

### CODE REVIEW

**Comment by HenriqueFSilva** [comment]
Would really like for this to be implemented!

File: pulsar-jms/src/main/java/com/datastax/oss/pulsar/jms/PulsarConnectionFactory.java [hunk 1]
```diff
@@ -70,26 +70,7 @@
...
-import org.apache.pulsar.client.api.PulsarClient;
-import org.apache.pulsar.client.api.PulsarClientException;
-import org.apache.pulsar.client.api.Reader;
-import org.apache.pulsar.client.api.ReaderBuilder;
-import org.apache.pulsar.client.api.Schema;
-import org.apache.pulsar.client.api.SubscriptionInitialPosition;
-import org.apache.pulsar.client.api.SubscriptionMode;
-import org.apache.pulsar.client.api.SubscriptionType;
-import org.apache.pulsar.client.api.TopicMetadata;
+import org.apache.pulsar.client.api.*;
```
Comment (eolivelli) [inline pulsar-jms/src/main/java/com/datastax/oss/pulsar/jms/PulsarConnectionFactory.java:73]: can you please remove this star import ?

**Review by eolivelli — CHANGES_REQUESTED** [review]
Hello,
Can you please rebase on top of the master branch ?
CI didn't run.

We are going to cut a new release soon and your improvement is very helpful, sorry for so late reply

**Comment by jffp113** [comment]
I changed the pull requesrt accordingly to your request.

**Pull request closed without merging**